src/std/env.ab

```diff
@@ -162,3 +162,13 @@ pub fun echo_error(message: Text, exit_code: Num = 1): Null {
     printf("\x1b[1;3;97;41m%s\x1b[0m\n", [message])
     if exit_code > 0 : exit(exit_code)
 }
+
+/// Returns whether or not Bash version is lower than provided version
+/// In a form of an array (ex. [3, 2] for Bash 3.2)
```

Comment:
It seems the parameters are two numbers, not an array. Are they the same?

Reply:
Ah my bad

Reply:
@lens0021 something like this sounds good?

```suggestion
/// In a form of an array (ex. bash_version_less_than(3, 2) for Bash 3.2)
```

Reply:
LGTM